### PR TITLE
Update licenses for 6.8.1 Release

### PR DESCRIPTION
**Pull request by vsethi09**

### CODE REVIEW

**Review by sumitjnn — APPROVED** [review]

**Review by iam-divya — APPROVED** [review]
LGTM

File: cdap-distributions/src/COPYRIGHT/org.mockito:mockito-inline:4.11.0/COPYRIGHT [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License
```
Comment (greeshmaswaminathan) [inline cdap-distributions/src/COPYRIGHT/org.mockito:mockito-inline:4.11.0/COPYRIGHT:1]: Test dependencies licenses need not be included in the licenses.

File: cdap-distributions/src/COPYRIGHT/junit:junit:4.13/COPYRIGHT [hunk 2]
```diff
@@ -0,0 +1,125 @@
+Eclipse Public License - v 1.0
```
Comment (greeshmaswaminathan) [inline cdap-distributions/src/COPYRIGHT/junit:junit:4.13/COPYRIGHT:1]: Leave out the licenses for test dependencies. We only need what is shipped in the release.

File: cdap-distributions/src/COPYRIGHT/com.puppycrawl.tools:checkstyle:6.11.12/COPYRIGHT [hunk 3]
```diff
@@ -0,0 +1,502 @@
+                  GNU LESSER GENERAL PUBLIC LICENSE
```
Comment (greeshmaswaminathan) [inline cdap-distributions/src/COPYRIGHT/com.puppycrawl.tools:checkstyle:6.11.12/COPYRIGHT:1]: Checkstyle licenses are not required.

File: cdap-distributions/src/COPYRIGHT/ant:ant:1.5/COPYRIGHT [hunk 4]
```diff
@@ -0,0 +1,49 @@
+/*
```
Comment (greeshmaswaminathan) [inline cdap-distributions/src/COPYRIGHT/ant:ant:1.5/COPYRIGHT:1]: ant license not required to be included

**Review by greeshmaswaminathan — COMMENTED** [review]
@vsethi09 , Only the shipped dependencies need the license, so ignore the test and build dependencies .

File: cdap-distributions/src/COPYRIGHT/org.mockito:mockito-inline:4.11.0/COPYRIGHT (hunk 1 above)
Comment (vsethi09) [reply cdap-distributions/src/COPYRIGHT/org.mockito:mockito-inline:4.11.0/COPYRIGHT:1]: As discussed, not a blocker.

File: cdap-distributions/src/COPYRIGHT/junit:junit:4.13/COPYRIGHT (hunk 2 above)
Comment (vsethi09) [reply cdap-distributions/src/COPYRIGHT/junit:junit:4.13/COPYRIGHT:1]: As discussed, not a blocker.

File: cdap-distributions/src/COPYRIGHT/com.puppycrawl.tools:checkstyle:6.11.12/COPYRIGHT (hunk 3 above)
Comment (vsethi09) [reply cdap-distributions/src/COPYRIGHT/com.puppycrawl.tools:checkstyle:6.11.12/COPYRIGHT:1]: As discussed, not a blocker.

File: cdap-distributions/src/COPYRIGHT/ant:ant:1.5/COPYRIGHT (hunk 4 above)
Comment (vsethi09) [reply cdap-distributions/src/COPYRIGHT/ant:ant:1.5/COPYRIGHT:1]: As discussed, not a blocker.

**Pull request merged**